soap/soap-tests.ts

```diff
@@ -4,7 +4,7 @@ import * as fs from "fs";
 import * as http from "http";
 
 const url = 'http://example.com/wsdl?wsdl';
-const wsdlOptions = { name: 'value' };
+const wsdlOptions = { name: 'value', httpClient: new soap.HttpClient() };
```

Comment:
found my mistake in my first commit.
( `s/httpRequest/httpClient/` )
Options are defined by https://github.com/vpulim/node-soap#options